antora-ui-camel/src/css/frontpage.css

```diff
@@ -50,24 +50,28 @@ header.frontpage a:active {
 
 .frontpage a.significant {
   margin: 1rem;
-  padding: 0.8rem 1.5rem;
-  background: var(--color-camel-orange);
-  color: var(--color-white);
+  padding: .8rem 1.5rem;
+  background: #ed8225;
+  color: #fff;
   border-radius: 8px;
   font-size: 1.2rem;
-  font-weight: bold;
+  font-weight: 700;
   outline: none;
   display: inline-block;
   white-space: nowrap;
+  box-shadow: 0 4px #8e480b;
```

Comment:
Buttons are designed without shadows, rebase and have a look at the current master or live website.

Reply:
I have pulled the fresh code now and checked the current website, there have been updates, current buttons have a shadow.